packages/react-instantsearch/examples/next-app/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "create-next-example-app",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "NODE_ENV=development next build",
```

Comment:
Shouldn't the build be NODE_ENV=production? Usually when hosting it provides better performance